parts/linux/cloud-init/artifacts/cni-telemetry.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Emit CNI telemetry
+
+[Service]
+Type=simple
+RemainAfterExit=true
+Environment="PYTHONUNBUFFERED=1"
+ExecStart=/usr/bin/python3 /opt/scripts/cni-telemetry.py
```

Comment:
I'm open to rewriting this in bash for consistency with other scripts if people feel strongly about it (I just happen to be more comfortable with Python, so I could get something up-and-running faster this way).